src/main/java/com/sparta/gourmate/domain/order/controller/OrderController.java

```diff
@@ -0,0 +1,80 @@
+package com.sparta.gourmate.domain.order.controller;
+
+import com.sparta.gourmate.domain.order.dto.OrderRequestDto;
+import com.sparta.gourmate.domain.order.entity.Order;
+import com.sparta.gourmate.domain.order.service.OrderService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/api/orders")
+public class OrderController {
+
+    private final OrderService orderService;
+
+    public OrderController(OrderService orderService) {
+        this.orderService = orderService;
+    }
+
+    // 주문 생성 (POST)
+    @PostMapping
+    public ResponseEntity<Order> createOrder(@RequestBody OrderRequestDto orderRequestDto) {
+        Order createdOrder = orderService.createOrder(orderRequestDto);
+        return new ResponseEntity<>(createdOrder, HttpStatus.CREATED);
+    }
+
+    // 주문 상세 조회 (GET) - 삭제되지 않은 주문만 조회
+    @GetMapping("/{orderId}")
+    public ResponseEntity<Order> getOrderById(@PathVariable UUID orderId) {
+        Order order = orderService.getOrderById(orderId)
```

Comment:
Entity 대신 Dto를 보내야 합니다
```suggestion
    public ResponseEntity<OrderResponseDto> getOrderById(@PathVariable UUID orderId) {
        OrderResponseDto order = orderService.getOrderById(orderId)
```